### DISCUSSION TITLE
How do i run photoshop2021install.sh in Bottles

### DISCUSSION THREAD

**Discussion by uJhV8xTzKBn5nv** [LinSoftWin/Photoshop-CC2022-Linux#105] [Q&A]
I want to install it in Bottles can some one point me in the right direction?

**Comment by DSDV** [comment]
Im am interested in that too :D

**Reply by MiMillieuh** [reply]
You can technically run it into bottles but the experience is way more buggy and crash way more.

And set it up is a pain.
